Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -33,3 +33,130 @@
     sprite: _CD/Clothing/OuterClothing/Coats/harmonyjacket.rsi
   - type: Clothing
     sprite: _CD/Clothing/OuterClothing/Coats/harmonyjacket.rsi
+
+# Europan Coats
+- type: entity
+  parent: ClothingOuterStorageBase
+  id: ClothingOuterCoatEuropanVet
+  name: europan jacket
+  description: A heavy leather jacket, winterized and waterproofed for harsh underwater environments.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/Europa/veterancoat.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/Europa/veterancoat.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
+        Heat: 0.90
+
+- type: entity
+  parent: ClothingOuterCoatEuropanVet
+  id: ClothingOuterCoatEuropanCoalition
+  name: coalition jacket
+  description: A heavy leather jacket, winterized and waterproofed for harsh underwater environments. Emblazoned with a Coalition patch on the shoulders.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/Europa/coalitioncoat.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/Europa/coalitioncoat.rsi
+
+- type: entity
+  parent: ClothingOuterCoatEuropanVet
+  id: ClothingOuterCoatEuropanSeparatist
+  name: separatist jacket
+  description: A heavy leather jacket, winterized and waterproofed for harsh underwater environments. Emblazoned with a Separatist patch on the shoulders.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/Europa/separatistcoat.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/Europa/separatistcoat.rsi
+
+# Cargo Gorka
+- type: entity
+  parent: ClothingOuterStorageFoldableBase
+  id: ClothingOuterCoatCargoBomber
+  name: cargo gorka
+  description: A heavy canvas gorka for cold shifts in Cargo.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/cargo_bomber.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/cargo_bomber.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
```

Comment:
You forgot to include the standard slash protection that winter coats get.

Any reason to not inherit from both ClothingOuterStorageFoldableBase and ClothingOuterWinterCoat?

Reply:
it can open and close, winter coat doesn't have that function

Reply:
You can inherit from both.

eg: 
```
parent: [ ClothingOuterStorageFoldableBase, ClothingOuterWinterCoat ]
```

Reply:
damn you for having good ideas, give me a moment